packages/server/graphql/types/RetroReflection.ts

```diff
@@ -50,6 +50,11 @@ const RetroReflection = new GraphQLObjectType<Reflection, GQLContext>({
       type: new GraphQLNonNull(new GraphQLList(new GraphQLNonNull(GraphQLID))),
       resolve: () => []
     },
+    groupColor: {
```

Comment:
any reason for denormalizing? since 1 reflection belongs to 1 retro phase item, i'd probably just keep the color on the retro phase item, although I could be missing something?

Reply:
Good catch! This actually isn't necessary. I added this when tracking down the `TypeError` caused by when there was a lack of reference to `groupColor` in `createReflection` I mentioned above. Fixed now.